### ISSUE TITLE
"not providing work fast enough" even immediately after receiving work

### ISSUE THREAD

**Issue by luke-jr** [luke-jr/bfgminer#49]
```
 [2012-08-02 10:57:38] LONGPOLL from pool 0 detected new block
 [2012-08-02 10:57:38] BFL0: Abandoning stale search to restart
 [2012-08-02 10:57:38] Pool 0 not providing work fast enough
```

**Comment by luke-jr** [comment]
Workaround in a6b9732 (2.7.0)

**Comment by luke-jr** [comment]
Seems to mostly be fixed in 2.7.5
